### PR TITLE
[relates #38] Datamapper separate feature

### PR DESCRIPTION
**Pull request by Stefan365**
correction of sf-to-db.feature
this is first step for fulfill task #38

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+/**
+ * Created by sveres on 11/29/17.
+ */
+public class IntegrationSaveOrAddStepConponent extends SyndesisPageObject {
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java]: Typo in component. And I've a feeling that this class hierarchy is too granular.

File: ui-tests/src/test/resources/features/integrations-sf-db.feature [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      # select postgresDB connection as 'to' point
+    Then Camilla is presented with the Syndesis page "Choose a Finish Connection"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL stored procedure" integration action
+    And she selects "add_lead" from "procedureName" dropdown
+#    And she stays there for "13000" ms
+    And clicks on the "Done" button
+
+      # add data mapper step
+    And she stays there for "3000" ms
```
Comment (dsimansk) [inline ui-tests/src/test/resources/features/integrations-sf-db.feature]: Is this still needed? Could we wait for some event to happen rather?

Comment (Stefan365) [reply ui-tests/src/test/resources/features/integrations-sf-db.feature]: @dsimansk generally I'm using it, but here forgotten to comment out.

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java (hunk 1 above)
Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java]: @dsimansk actually "syndesis-integrations-save-or-add-step" is only on the page "Add to Integration" page. Yet it is not child of "syndesis-integrations-connection-select" but only of "syndesis-integrations-edit-page" (main page). If I want to check presence via our standardized way (SyndesisPage), I think I don't have other option as to have separate component class for it.

Comment (dsimansk) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java]: Hmm, ok then. But pls fix the typo in the class name :)

Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java]: @dsimansk done.

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java [hunk 3]
```diff
@@ -292,7 +292,7 @@ public void fillActionConfigureField(String fieldId, String value) {
 	 * @param position
 	 */
 	@Then("^she is prompted to select a \"([^\"]*)\" connection from a list of available connections$")
-	public void verifyTypeOfConnection(String position) {
+public void verifyTypeOfConnection(String position) {
```
Comment (lvydra) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java]: Only small correction.

Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java]: @lvydra done

**Pull request merged**